### ISSUE TITLE
Incomplete DB table names when using DB prefix

### ISSUE THREAD

**Issue by kfriend** [mailchimp/mc-magento2#113]
When Magento is configured to use a DB prefix (e.g. `mage_`), this extension's DB scheme migrations will fail. It seems that the usage of the `getTableName()` method, when calling directly on a connection, will not include the DB prefix. This issue (https://github.com/magento/magento2/issues/6402) suggest that calling `getTableName()` directly on the resource, instead of connection, will resolve the issue.

SQL error when running `bin/magento setup:upgrade`

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'the_db.sales_order' doesn't exist, query was: DESCRIBE `sales_order`
```

Thanks

**Comment by gonzaloebiz** [comment]
What version of m4m2 do you have?

**Comment by kfriend** [comment]
Thanks @gonzaloebiz. I was using 1.0.24 from the Magento store. Just removed and installed via Composer, which gave me 1.0.25. I see the note about the DB prefix fix in 1.0.25's release notes.

Closing this issue. Sorry about wasting time 😊
